### PR TITLE
Add satysfi-assert-eq

### PR DESCRIPTION
**Pull request by puripuri2100**
Add [`satysfi-assert-eq.0.1.0`](https://github.com/puripuri2100/SATySFi-assert-eq).

"Provides function as the Rust's assert_eq! macro"

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ env:
     - >-
       PACKAGES_FOR_STABLE="
       satysfi-base
+      satysfi-assert-eq
```
Comment (na4zagin3) [inline .travis.yml]: can you please move this line above satysfi-base to order the list alphabetically?

**Comment by puripuri2100** [comment]
Why!?

**Comment by puripuri2100** [comment]
`curl error code 429` has occurred

**Comment by na4zagin3** [comment]
I'm rerunning the job…. There may be the room of improvement in build scripts...

**Comment by na4zagin3** [comment]
Merged. Thanks!

**Pull request merged**